sdk/spring/azure-spring-integration-servicebus/src/main/java/com/azure/spring/integration/servicebus/ServiceBusTemplate.java

```diff
@@ -37,14 +42,17 @@
     protected CheckpointConfig checkpointConfig = CHECKPOINT_RECORD;
     protected ServiceBusClientConfig clientConfig = ServiceBusClientConfig.builder().build();
     protected ServiceBusMessageConverter messageConverter;
+    protected InstrumentationManager instrumentationManager;
 
-    public ServiceBusTemplate(@NonNull T senderFactory) {
-        this(senderFactory, DEFAULT_CONVERTER);
+    public ServiceBusTemplate(@NonNull T senderFactory, InstrumentationManager instrumentationManager) {
```

Comment:
I don't like this way of constructing the template by requiring a InstrumentationManager.